client/src/services/LoginService.js

```diff
@@ -1,50 +1,49 @@
+import ErrorMessage from 'models/ErrorMessages';
+
 export default class LoginService {
   static login = (data) => {
+
     const formData = new FormData();
 
     formData.append('username', data.username);
     formData.append('password', data.password);
 
-
-    // Configure POST request options
-    const fetchOptions = {
```

Comment:
Instead of calling it fetchOptions and then describing it as Post request options, we can rename it as postRequestOptions and it will be self-descriptive.